src/content/api/module-variables.md

```diff
@@ -137,6 +137,13 @@ The internal chunk loading function. Takes two arguments:
 Access to the internal object of all modules.
 
 
+### `__webpack_get_script_filename__` (webpack-specific)
+
+This is a function that, when called with a chunk id, returns the filename of that chunk.
+
```

Comment:
```suggestion
```

Reply:
This is newline is necessary for `T>` to work.